### PR TITLE
hw4 belal alrammah

### PR DESCRIPTION
**Pull request by iBelal995**
every part indicated with comment

### CODE REVIEW

File: HW4/src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+//Part 1 Null safty
+//fun getLength(str: String?) : Int {
+//    if (str != null)
+//    return str.length
+//    else
+//        return 0
+//}
+//
+//fun main(args: Array<String>) {
+//    println(getLength(null))
```
Comment (MohamedFouad96) [inline HW4/src/main/kotlin/Main.kt:10]: Change this to take dynamic argument not static

File: HW4/src/main/kotlin/Main.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+//    val menuData = "The quick brown fox jumps over the lazy dog"
+//    val data = menuData.split(' ')
+//    val new = "${data[2]} ${data[3]} ${data[4]}"
+//    println(new)
+//}
+//Part 3 numbers
+//fun main(){
+//    println("Write the temperature in F° to convert it to C°:")
+//   var F = readLine()!!.toInt()
+//    var C = (F-32)*0.556
```
Comment (MohamedFouad96) [inline HW4/src/main/kotlin/Main.kt:23]: this is not the correct equation, try this `C = (5 (F-32)) / 9`

Comment (iBelal995) [reply HW4/src/main/kotlin/Main.kt:23]: it's actually the same I just make it simple .

Comment (MohamedFouad96) [reply HW4/src/main/kotlin/Main.kt:23]: okay its work correctly😁

**Comment by MohamedFouad96** [comment]
Excellent work belal

**Pull request closed without merging**